css/loginStyles.css

```diff
@@ -0,0 +1,69 @@
+body {
+    background-color: #406bb7;
+}
+
+.bg {
+    width: 95%;
+    background-color: #ffffff;
+    border: solid 5px #406bb7;
+    border-radius: 12px;
+    margin: 0 auto;
+}
+
+.title, .title2 {
+    background-image: url("../images/pokebees.jpg");
```

Comment:
Slightly confusing to have he Pokebees part as a background for both and then overwriting later. I'd  separate.

Reply:
done